stats.lua

```diff
@@ -392,6 +402,54 @@ local function append_display_sync(s)
 end
 
 
+local function append_filters(s, prop, prefix)
+    local length = 0
+    local filters = {}
+
+    local speed = mp.get_property_osd("speed")
+    if speed ~= "1.00" then
```

Comment:
Still, it's not a filter and as such it has no place here. Especially not twice.
And as said in other issues, this script is not intended to echo the options a user chose. If a user is setting speed to `x` I don't see any reason to echo this information here as the user did that deliberately and knows about it. Filters as a whole, as also mentioned, are a bit different as some can be inserted without the user knowing it or doing it himself, so this information is justified.